.github/workflows/build.yaml

```diff
@@ -41,32 +47,33 @@ jobs:
             arch: amd64
           - name: aarch64-pc-windows-msvc
             arch: arm64
+    
+    runs-on: ${{ matrix.runner.name }}
+
+    env:
+      # WDK 10.0.22621 installed via winget channel on ARM64 runner doesn't ship certmgr and makecert binaries
```

Comment:
I think at this point we've validated that this is not true. it is that the SDK is not installed for that version

Reply:
Updated the comment.

Reply:
Installing 10.0.22621 SDK with installer works now. We no longer need this condition and removed it in the latest commit.